Develop

simondpowell opened the pull request:
Added clipart and material images.
Updated after Samson's feedback.

simondpowell committed:
Changed first page title

simondpowell committed:
Added clipart and material images

simondpowell committed:
Test

simondpowell committed:
Delete Test note

samsonfsb commented:
Awesome stuff dude!!

Minor changes and then I reckon we are good to go!!

**Slide 4:** 
- Quantity column could be a bit clearer i.e. it is not immediately obvious that the numbers are quantity
- Angle iron pic needs to be a bit closer up as it might not be clear what that is

**Slide 5**
- No dimension for thickness of PVC pipe shown in drawing

**Slide 8**
- This slide is fantastic

**Slide 10**
- For some reason the lines are super thick in the left hand diagrammes so it is a little unclear what these are showing. It looked better in a previous version.

**Slide 12**
- Could use the line "using a protractor" or similar

**Slide 16**
- Nail polish apparently pretty good thread lock
- Is thread lock listed as a material?

simondpowell committed:
Added photos of materials, changed blade profile cut order, added PDF export image to show comparison

simondpowell commented:
Addressed some of your problems. The slide numbers you stated didn't line up to the slides you were commenting about. Not sure if this was a mistake or what else could've caused this? Not a massive issue but I wasn't 100% sure on which images you were commenting on every time.

Quantity column header is removed. Added x in front of each number i.e. x2, x3 etc so hopefully this is more clear.

Angle iron view updated and also take a look at materials images. I think this is what you wanted to me to anyway but I didn't beause some of the images online look poo (maybe I need to take my own). I do like doing it this way though and then I think every time a new part is introduced in the instructions we give it a label as I have done for the mostpart.

Haven't added PVC thickness because it's dependent on diameter, i.e. user has no control over it, and it was difficult to show on an isometric view. Can add front view in easily if you think I should?

Re: Slide 8 - not sure which one this is but thanks!

As for thick lines - again I'm not sure which image you mean in particular but a common theme is loss of quality due to thick lines and pixelated text. Thick lines are something that Onshape has specifically done for one reason or another and it's not yet possible to change line style. Once exported to PDF lines look a lot sharper and this is shown in the second to last slide next to the Onshape image for comparison. Once we're happy with all images I can export and redo but I still think we'll get the same issues when having to show fine details along side large parts. A work around is to make those fine details less fine - easily done with threads but maybe not so easy for other things. The way to export these images while maintaining quality is definitely something we need to figure out.

Added 'use protractor'.

Added thread lock as material. Wasn't sure about whether this is a material or a tool, or neither. Guess it's not a huge issue.

samsonfsb commented:
> Addressed some of your problems.

Hahaha. Engineer by day, psychotherapist by night.

Awesome work! See my pull request for a final change I would make

simondpowell committed:
Line styles improved by exporting as PDF and screenshotting. Minor changes to layout. Photos of tools and materials included.

simondpowell committed:
All images exported from PDF to show better line detail
